xray/xray-sampling.tf

```diff
@@ -0,0 +1,18 @@
+# Refer to here for the sampling rules
```

Comment:
what do we need this for?

Reply:
Do u mean the comments or the files?

Reply:
Do you refer to the comments? 
Leave a note here so we have a docs reference of the sampling rules. Not a really important notes actually

Reply:
ah, so this is excluding traces for the health check endpoints, good thinking!